eth/filters/dropped_tx_subscription.go

```diff
@@ -2,23 +2,74 @@ package filters
 
 import (
 	"context"
+	"time"
 
-	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/core"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/internal/ethapi"
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
 type dropNotification struct {
-	TxHash common.Hash `json:"txhash"`
-	Reason string      `json:"reason"`
+	// TxHash common.Hash `json:"txhash"`
```

Comment:
Dead code?

Reply:
Austen left this commented but sure I can delete

Reply:
Nah, leave 'em.  🙂